packages/generators/generators/constants.js

```diff
@@ -0,0 +1,50 @@
+import packageExists from '../utils/packageExists.js';
+
+export const prompts = [
+	{
+		type: 'input',
+		name: 'name',
+		message: 'What should it be called?',
```

Comment:
I'd rephrase this to something like:

```suggestion
		message: 'What should the package be called?',
```

There's a substantial improvement when we use concrete words instead of "it", "this" or "that" when explaining / describing things in software development. Tanya Reilly suggests this also in the book [The Staff Engineer's Path](https://www.oreilly.com/library/view/the-staff-engineers/9781098118723/).